Composer/packages/client/src/pages/language-generation/index.tsx

```diff
@@ -113,15 +126,15 @@ const LGPage: React.FC<RouteComponentProps> = props => {
 
   function onSelect(id) {
     if (id === '_all') {
-      navigateTo('/language-generation');
+      navigateTo('/language-generation/');
     } else {
       navigateTo(`language-generation/${id}`);
     }
   }
 
   function onToggleEditMode() {
     setEditMode(!editMode);
-    setCodeRange(null);
+    setCodeRange(undefined);
```

Comment:
why the need for this change?

Reply:
The codeRange' type was defined in lgEditor as Partial<ICodeRange> | -1 | undefined. So I use undefined to replace null.